report/Evaluation/Essence.tex

```diff
@@ -0,0 +1,56 @@
+
+Regarding the SWOT method, it gave meaning to use this method in Ecology. 
+It is important to know if we are using the right people or services.
+One thing that helped us, was that we had written down a small piece of text to each element, in regards to SWOT, and this made it easy for us to score them, because we could, rather fast, see what the pros and cons were in a given element.
+If we had not done this, this process might have been more challenging, because we needed to remember what was the pros and cons, and we needed to be on the same page, in terms of our understanding of the element in question.
+
+Overall we liked the process in Ecology, and it was fun to give the different elements points, in regards to the SWOT method. 
+Here we had some good discussions on why we though a given element, had the strength it had and so forth.
+% \\TODO Find out if this sentense needs to be here or not.
+As we were done with this process, we came to the conclusion that it will be easier to understand the difference between the boxes, and therefore you can identify points faster, because you have this understanding of Essence, after utilizing Essence.
+
+The key thing to take away is that the lines between the different External elements, needs to be "clearer", and that External artifacts and services could be merged together.
+
+\subsection{Leverage}
+% \\TODO Find better wording in the end of this sentense 
+This process follows the same principles as Ecology, therefore many of the things said before, is also here.
+Overall it was easy to find the internal technologies, however, we had a hard time finding points to the other internal boxes.
+
+In our discussions, we had problems seeing what the difference between the different boxes is, in Ecology and Leverage.
+Somewhere easy to spot, but some, a little harder to see.
+One solution to this is to be more specific when talking about the different boxes, both in External and Internal and have more in-depth examples, which can help the reader to understand the terms better.
+
+In regards to the evaluation of the different Leverage Points, we opted to use the PCRT method, to rank and describe each point.
+The method itself was easy to use and work with, however, one problem we had, was that it was hard to distinguish all points, in terms of, if it is about the technology in regards to the project, or if it is only about the technology.
+We might have ranked the Leverage Points differently if we only looked at the technology itself and not looked at it in the context of the project.
+Therefore it could be helpful if there was a clear point about this.
+
+In Leverage, the process of creating a list could be helpful here.
+
+The key thing to take away in Leverage is to have a more clear line in what, which points should be in Ecology, e.g. External and which ones in Leverage, e.g. Internal, and have a more clear line in how to rank the Leverage Points using the PCRT method.
```

Comment:
```suggestion
Our key critique regarding Leverage is that there is too much ambiguity, and it would be beneficial to try and eliminate this. 
If possible a methodical approach to determining whether a point, either External or Internal, is an Ecology or Leverage would be beneficial.
Also, guidance on how to rank the Leverage Points using the PCRT method would help.
```

Dont know if I understood what you where trying to say correctly, but here is my suggestion ;D